sql/core/src/test/scala/org/apache/spark/sql/jdbc/JDBCV2Suite.scala

```diff
@@ -154,6 +154,18 @@ class JDBCV2Suite extends QueryTest with SharedSparkSession with ExplainSuiteHel
       conn.prepareStatement("INSERT INTO \"test\".\"dept\" VALUES (1, 1)").executeUpdate()
       conn.prepareStatement("INSERT INTO \"test\".\"dept\" VALUES (2, 1)").executeUpdate()
 
+
+      conn.prepareStatement(
+        "CREATE TABLE \"test\".\"employee1\" (dept INTEGER, name TEXT(32), " +
```

Comment:
Why not use `employee` ?

Reply:
employee table does not have two string columns that can be used to test the concat function

Reply:
You can cast other column to string.

Reply:
ok, I thought it didn't support cast pushdown yet.

Reply:
done